processor/src/main/java/com/linecorp/decaton/processor/runtime/SubscriptionBuilder.java

```diff
@@ -126,6 +131,16 @@ public SubscriptionBuilder enableRetry(RetryConfig config) {
         return this;
     }
 
+    /**
+     * @param tracingProvider Add tracing provider that will be called to trace the execution of processing
+     * each record.
+     * @return updated instance of {@link SubscriptionBuilder}.
+     */
+    public SubscriptionBuilder enableTracing(TracingProvider tracingProvider) {
+        this.tracingProvider = tracingProvider;
```

Comment:
please filter this with `Objects.requireNonNull` since null argument for `enableXX` just doesn't make sense.